contracts/contracts/MerkleGrantRoundPayout.sol

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.5;
+
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/utils/cryptography/MerkleProof.sol";
+
+/**
+ * @notice The MerkleGrantRoundPayout contract eligble grant owners to claim their
+ * match after round ends and funds have been loaded into the contract
+ * It allows grant owners to claim their funds in the given ERC20
+ * token upon verif merkleRoot
+ *
+ * This contract does the following
+ *  - funds transferred from GrantRound contract
+ *  - merkle root which contains how the funds are segrated and who can claim them
+ *  - emit Claimed event when grant owner has claimed funds
+ *
+ * @dev code sourced from https://github.com/Uniswap/merkle-distributor/blob/0d478d722da2e5d95b7292fd8cbdb363d98e9a93/contracts/MerkleDistributor.sol
+ * Changes made:
+ *  - renamed isClaimed to hasClaimed
```

Comment:
Just curious why this rename? Also recommend using backticks around variable method names just for clarity, e.g. `` `account` renamed to `payee` ``

Reply:
language-wise it seemed more right to say the contract has a function that allows to check if a user `hasClaimed` funds.
I can switch it back if needed

Reply:
Totally fine with me, just curious!